chromium_src/chrome/browser/flags/android/chrome_feature_list.cc

```diff
@@ -4,6 +4,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/common/brave_features.h"
+#include "brave/components/brave_wallet/common/features.h"
```

Comment:
Should the additions in there be under `#if BUILDFLAG(BRAVE_WALLET_ENABLED)`?

Reply:
Also, needs `#include "brave/components/brave_wallet/common/buildflags/buildflags.h"` now before using `#if BUILDFLAG(BRAVE_WALLET_ENABLED)`.

The `#if BUILDFLAG(BRAVE_WALLET_ENABLED)` include typically should go underneath other includes separated from them by an empty line.

Reply:
yeah, my bad, missed that and forgot to try to build it locally